firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/report/ReportUploader.java

```diff
@@ -96,14 +99,21 @@ public boolean uploadReport(Report report, boolean dataCollectionToken) {
       final CreateReportRequest requestData =
           new CreateReportRequest(organizationId, googleAppId, report);
 
-      final boolean sent = createReportCall.invoke(requestData, dataCollectionToken);
+      boolean sent;
+      // For now, send native reports to reports endpoint regardless of the setting.
```

Comment:
Worth a TODO?

Reply:
I'm going to circle back to this in the next week or two, but happy to add a TODO if you think it's worth it.

Reply:
I like shorter TODOs better than longer ones because those are the ones we actually get to address, but happy to punt on it for this PR and just do it in future cases if it saves you work :)